dask_ml/model_selection/_incremental.py

```diff
@@ -118,7 +127,10 @@ def _fit(
     fit_params=None,
     scorer=None,
     random_state=None,
+    verbose=False,
 ):
+    _show_msg("[CV] train examples = {}".format(len(y_train)), verbose)
```

Comment:
With logging you typically use `%`-style formatters, and pass the values in. This skips the formatting step if it's not needed.

```
logger.info([CV] train examples = %d, len(y_train))
```